.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+name: Main workflow (PR)
+
+on: [pull_request]
```

Comment:
Note that this will only run for pull requests coming from forks, not from branches opened on this repo (like this PR).

This is what I use to cover all cases:

```yaml
on:
  push:
    branches:
      - master
  pull_request:
    branches:
      - master
```

Reply:
@jparise Thank you, updated